test/lit/wasm-emscripten-finalize/em_js.wat

```diff
@@ -3,20 +3,32 @@
 
 ;; RUN: wasm-emscripten-finalize %s -S | filecheck %s
 
-;; Both functions should be stripped from the binary
+;; All functions should be stripped from the binary, regardless
+;; of internal name
 ;; CHECK-NOT:  (func
 
+;; The data section that contains only em_js strings should
+;; be stripped.
+;; CHECK-NOT: (i32.const 512) "Only em_js strings here\00")
```

Comment:
Would this change correctly test that the entire segment is stripped, rather than just changed?
```suggestion
;; CHECK-NOT: (i32.const 512)
```